R/simulate_infections.R

```diff
@@ -1,19 +1,205 @@
-#' Deprecated; use [forecast_infections()] instead
+#' Simulate infections using the renewal equation
 #'
-#' Calling this function passes all arguments to [forecast_infections()]
-#' @description `r lifecycle::badge("deprecated")`
-#' @param ... Arguments to be passed to [forecast_infections()]
-#' @return the result of [forecast_infections()]
+#' Simulations are done from given initial infections and, potentially
+#' time-varying, reproduction numbers. Delays and parameters of the observation
+#' model can be specified using the same options as in [estimate_infections()].
+#'
+#' In order to simulate, all parameters that are specified such as the mean and
+#' standard deviatoin of delays or observation scaling, must be fixed.
+#' Uncertain parameters are not allowed.
+#'
+#' A previous function called [simulate_infections()] that simulates from a
+#' given model fit has been renamed [forecast_infections()].
```

Comment:
- Should we provide a quick justification here?
- Will this statement be removed after some time?

Reply:
Have added some context in cf0b772a